README.md

```diff
@@ -207,10 +208,11 @@ The `NuxtAuthHandler` accepts [all options that NextAuth.js accepts for its API
 ##### Example with two providers
 
 Here's what a full config can look like, wee allow authentication via a:
-- Github Oauth flow,
+- Github Oauth flow
 - a username + password flow (called `CredentialsProvider`)
+- Strapi JWT example with the `CredentialsProvider`
```

Comment:
wrong section -> in this section we only have the GH oauth flow and the credentials flow, the strapi flow is below